source/command-monitoring/tests/command.json

```diff
@@ -7,6 +7,7 @@
   ],
   "collection_name": "test",
   "database_name": "command-monitoring-tests",
+  "read_preference": "primary",
```

Comment:
Drivers are supposed to ignore primary read preference for the purposes of wrapping a command or query filter in $query, so this should be a no-op.  Should use primaryPreferred, as Bernie suggested.